packages/mds-geography/api.ts

```diff
@@ -89,7 +98,22 @@ function api(app: express.Express): express.Express {
           )
         }
 
-        const geographies = summary ? await db.readGeographySummaries(params) : await db.readGeographies(params)
+        const options = { includeGeographyJSON: !summary }
+        const geographies = await (() => {
+          if (params.get_unpublished) {
+            if (params.get_published) {
+              throw new BadParamsError('cannot have get_unpublished and get_published both be true')
+            } else {
+              return GeographyServiceClient.getUnpublishedGeographies(options)
+            }
+          } else {
+            if (params.get_published) {
+              return GeographyServiceClient.getPublishedGeographies(options)
+            } else {
+              return GeographyServiceClient.getGeographies(options)
+            }
+          }
+        })()
```

Comment:
This IIFE smells of Neil 😄 ... is there good reason not to make this a separate function?
```ts
const geographies = await getGeographies(params, options)
```

Reply:
Neil did recommend it yes